partitura/musicanalysis/voice_separation.py

```diff
@@ -962,7 +993,9 @@ def estimate_voices(self):
         maximal_contigs_idxs = np.where(self._voices_per_contig == self.num_voices)[0]
 
         # indices of the non maximal contigs
-        non_maximal_contigs_idx = np.where(self._voices_per_contig != self.num_voices)[0]
+        non_maximal_contigs_idx = np.where(self._voices_per_contig != self.num_voices)[
```

Comment:
The way to solve this would be either shorter variable names (may not be best) e.g.:
```
non_max_contigs_idx = np.where(self._voices_per_contig != self.num_voices)[0]
```
or split into multiple variables to shorten lines (better I think):
```
non_maximal_contigs = self._voices_per_contig != self.num_voices
non_maximal_contigs_idx = np.where(non_maximal_contigs)[0]
```

Reply:
Fixed as proposed in commit  e64531d. Also, split line 993 to ensure a max line length of 79 characters.